dashboard/test/integration/omniauth/facebook_test.rb

```diff
@@ -119,17 +119,6 @@ class FacebookTest < ActionDispatch::IntegrationTest
       )
     end
 
-    test 'user_type is usually unset on finish_sign_up' do
```

Comment:
With no more references to `/users/finish_sign_up`, these tests are no longer needed.